core/src/main/java/hudson/model/Computer.java

```diff
@@ -1040,6 +1040,16 @@ public final long getDemandStartMilliseconds() {
     }
 
     /**
+     * Returns the {@link Node} description for this computer
+     */
+    @Exported
+    public @Nonnull String getDescription() {
+        Node node = getNode();
+        return (node != null) ? node.getNodeDescription() : "";
```

Comment:
@oleg-nenashev Please could you explain to me why do you think is better to return null instead of an empty string?Thank you in advance.